### ISSUE TITLE
lint: remove go-tools/cmd/unused linter workarounds for Context.require field

### ISSUE THREAD

**Issue by quasilyte** [go-critic/go-critic#614]
In order to pass CI checks, this code was added (https://github.com/go-critic/go-critic/pull/613/commits/a0585388825a6d6e5007fe4ff629cc807c6e5401):
```go
var _ = Context{}.require
```
This is intended to make `unused` linter convinced that `require` field is used.

Should be removed as soon as we can pass tests without it.

----

Probably related issues:
https://github.com/dominikh/go-tools/issues/311
https://github.com/dominikh/go-tools/issues/290
I haven't tried to nail down what precisely causes it to decide that this field is unused,
but it's probably because of somewhat complicated flow in `lint` package (it also uses `reflect` package).

**Comment by cristaloleg** [comment]
Closing, looks like an old fixed thing.
